### PR TITLE
Update direct-routing-sbc-multiple-tenants.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2575

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-sbc-multiple-tenants.md [hunk 1]
```diff
@@ -199,37 +199,35 @@ For example: test@sbc1.customers.adatum.biz
...
 •	A derived trunk, that does not require registration. It is simply a desired host name added in from of the carrier trunk. It derives all of its configuration parameters from the carrier trunk. The derived trunk doesn't need to be created in PowerShell, and the association with the carrier trunk is based on the FQDN name (see details below).
 
-Provisioning logic and example.
+Provisioning logic and example:
 
-•	Carriers only need to set up and manage a single trunk  (carrier trunk in the carrier domain), using the Set-CSOnlinePSTNGateway command. In the example above it is adatum.biz;
-•	In the customer tenant, the carrier need only to add the derived trunk FQDN to the voice routing policies of the users. There is no need to run New-CSOnlinePSTNGateway for a trunk.
-•	 The derived trunk, as the name suggests, inherits or derives all the configuration parameters from the carrier trunk. 
+•	Carriers only need to set up and manage a single trunk  (carrier trunk in the carrier domain), using the Set-CSOnlinePSTNGateway command. In the example above it is adatum.biz;</br>
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-sbc-multiple-tenants.md]:
```suggestion
•	Carriers only need to set up and manage a single trunk (carrier trunk in the carrier domain), using the Set-CSOnlinePSTNGateway command. In the previous example it is adatum.biz;</br>
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
The changes in this pull request are not consistent.  It looks like cmdlet syntax was changed in some instances, but not others.  Closing until this is resolved.

**Comment by dariomws306** [comment]
Hi @CarolynRowe, should we create a new PR? There is a typo in this article and also format doesn't look good at the end.

Thanks.

**Comment by CarolynRowe** [comment]
Yes, please.  Is some cases, the PR changes the capitalization of the cmdlet syntax, but only in one place, so it’s introducing inconsistency.

**Comment by dariomws306** [comment]
Done:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/2853

**Pull request closed without merging**